<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert</class>
 <widget class="QMainWindow" name="insert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>სახელი</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_surname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>გვარი</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_age">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ასაკი</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gender">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>გენდერი</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_grade">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ნიშანი</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_insert">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ჩაწერე</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_insert_status">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_surname">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_gender">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_age">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_grade">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
